website/content/docs/agent/telemetry.mdx

```diff
@@ -684,3 +684,22 @@ agent. The table below describes the additional metrics exported by the proxy.
 | `consul.proxy.web.upstream.conns`   | Shows the current number of connections open from a proxy instance to an upstream. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to. | connections | gauge   |
 | `consul.proxy.web.inbound.rx_bytes` | Increments by the number of bytes received from an upstream connection. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to.       | bytes       | counter |
 | `consul.proxy.web.inbound.tx_bytes` | Increments by the number of bytes transferred to an upstream connection. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to.      | bytes       | counter |
+
+## Peering metrics
+
+** Requirements: **
+* Consul 1.13.0+
+
+Consul 1.13 introduced [cluster peering](/docs/connect/cluster-peering). These metrics are only emitted by the leader server.
+
+| Metric                                | Description                                                           | Unit   | Type    |
+| ------------------------------------- | ----------------------------------------------------------------------| ------ | ------- |
+| `consul.peering.exported_services`    | Counts the number of services exported to a peer cluster.  | count  | gauge   |
+
+### Labels
+
+| Label Name                            | Description                                                            | Possible values                            |
+| ------------------------------------- | ---------------------------------------------------------------------- | ------------------------------------------ |
+| `peer_name`                           | The name of the peering on the reporting cluster/ leader.              | Any defined peer name in the cluster.      |
+| `peer_id`                             | The id of the peering on the reporting cluster/ leader.                | Any UUID.                                  |
```

Comment:
```suggestion
| `peer_id`                             | The ID of the peering on the reporting cluster/ leader.                | Any UUID.                                  |
```
Not that anyone would be confused that we're talking about the psychoanalytical id